docs/en/operations/settings/settings.md

```diff
@@ -3285,3 +3285,30 @@ Possible values:
 -   1 — The `LowCardinality` type is converted to the `DICTIONARY` type.
 
 Default value: `0`.
+
+## materialized_postgresql_max_block_size {#materialized-postgresql-max-block-size}
```

Comment:
btw, I am not sure whether it is better in this file and not in materailized_postgresql.md becuase these are not settings like all other here, - i.e. not added via configuration file, but they are only table-settings which are added via special syntax CREATE TABLE ... SETTINGS ... and can be applied to only this single table. But may be its ok.

Reply:
@sevirov, @kssenii, Yes, these settings should not be here, because it's database engine settings, not profile/user/session/query settings. Let's move them to `materialized-postgresql.md`